components/tableNodes/NodeTable.component.tsx

```diff
@@ -122,10 +104,10 @@ export function NodeTable({ show }: NodeTableProps): JSX.Element {
                   {}
                 </TableCell>
                 <TableCell component="th" scope="row">
-                  {check_amount_msat(channel.amount_msat)}
+                  {channel.channel_total_sat + " sats"}
```

Comment:
Just a question here, in this case `channel_tot_sat` is taken from what command?

I do not see it in the docs https://lightning.readthedocs.io/lightning-listfunds.7.html

Reply:
for `listfunds` we have `channel_total_sat` but in the docs it is not mentioned

![image](https://user-images.githubusercontent.com/24366008/181829856-27026ef2-bb19-4f3b-b2e4-91af0b430c66.png)

Reply:
ok, maybe we need to run the aws node withe API deprecated set to false, I search inside the code and looks like that it is a deprecated stuff https://github.com/ElementsProject/lightning/blob/e70729b04befe44d603370b96fbb10dc00bd342d/wallet/walletrpc.c#L358

I think we need to use the `amount_msat` here and we need also a check over a string, because in 6 months this value will became a int value